config/env_sample.json

```diff
@@ -128,8 +128,11 @@
         "CACHE_KEY_PREFIX": "myla",
         "CACHE_OPTIONS": {"COMPRESS_MIN_LENGTH": 5000, "COMPRESS_LEVEL": 6}
     },
+    "/*  Set this to true to enable the internal (Django) Login methods.": "*/",
+    "/*  This is enabled by default if no external auth (Currently LTI/SAML) is set.": "*/",
+    "ENABLE_BACKEND_LOGIN": false,
```

Comment:
I feel this needs to be added to the LTI wiki page.

Reply:
Okay I updated this